tests/app/main/views/service_settings/test_service_settings.py

```diff
@@ -2306,10 +2306,10 @@ def test_resume_service_after_confirm(
         ),
     )
 
-    assert mock_api.called_once_with(
+    mock_api.assert_called_once_with(
         "/service/{}/resume".format(SERVICE_ONE_ID), data=None
```

Comment:
I've been trying to update all the "".format() over to f strings when possible, for clarity and simplification, as well as idiomatic python purposes, could this line be changed to that?

`f"/service/{SERVICE_ONE_ID}/resume", data=None`

Reply:
This should work with f"/service/{SERVICE_ONE_ID}/resume" . I will update.